examples/plug-and-play/demo_PnP_custom_optim.py

```diff
@@ -0,0 +1,226 @@
+"""
+PnP with custom optimization algorithm (Condat-Vu Primal-Dual)
+====================================================================================================
+
+This example shows how to set its own custom optimization algorithm. 
+For example, here, we implement the Condat-Vu Primal-Dual algorithm for Single Pixel Camera (SPC) reconstruction.
```

Comment:
Maybe here we should explain a little bit more, that the only thing that the user has to do is define an iterator.